cart/cart.html

```diff
@@ -0,0 +1,200 @@
+                        <div class="right-part">
+                            <div class="card-title">
+                                Heading
+                            </div>
+                            <div class="card-text">
+                                This is product
+                            </div>
+                            <div class="price">
+                                <div class="discounted-price">
+                                    Rs.250
+                                </div>
+                                <div class="actual-price">
+                                    <s>Rs.300</s>
+                                </div>
+                                <div class="discount">
+                                    (10%)
+                                </div>
+                            </div>
+                            <div class="card-footer">
+                                <button class="add-cart-button">Add To Cart</button>
+                                <button class="add-cart-button">Add To Wishlist</button>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="horizontal-card">
+                    <div class="card">
+                        <div class="left-part">
+                            <div class="card-image">
+                                <img class="image" src="../images/1480051080Camlin Artist's oil colours.jpg" alt="">
+                            </div>
+                        </div>
+                        <div class="right-part">
+                            <div class="card-title">
+                                Heading
+                            </div>
+                            <div class="card-text">
+                                This is product
+                            </div>
+                            <div class="price">
+                                <div class="discounted-price">
+                                    Rs.250
+                                </div>
+                                <div class="actual-price">
+                                    <s>Rs.300</s>
+                                </div>
+                                <div class="discount">
+                                    (10%)
+                                </div>
+                            </div>
+                            <div class="card-footer">
+                                <button class="add-cart-button">Add To Cart</button>
+                                <button class="add-cart-button">Add To Wishlist</button>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+            <div class="bill">
+                <div class="spaced-list">
+                    <div class="list-title">
+                        price Details
+                    </div>
+                    <li>
```

Comment:
`li` elements always need a parent `ul` and `ol`. This is not the best practice because screen readers won't announce li if there is no parent ul and ol

Reply:
made the changes